controllers/openstackcluster_controller.go

```diff
@@ -268,16 +268,15 @@ func reconcileNormal(ctx context.Context, log logr.Logger, client client.Client,
 	if err != nil {
 		return ctrl.Result{}, err
 	}
+
+	if len(openStackCluster.Spec.ControlPlaneAvailabilityZones) == 0 {
+		log.Info("There is openStackCluster.Spec.ControlPlaneAvailabilityZones provided, this can cause issue if the administrator credential is used because `internal` availability zone will be a candidate")
```

Comment:
This is user-facing information and the logs are for devs so I'm not sure the right person would ever see this. I'd omit it personally. The docs message has it covered.

Reply:
ok